src/Utils/FileExport.php

```diff
@@ -182,14 +182,23 @@ protected function getRowCells(Model $model, array $args, stdClass $row): array
                             $property = $column['column'];
                             $value = $edge->_node->$property ?? null;
                         }
-                        $cells[] = WriterEntityFactory::createCell($value);
+                        $cells[] = WriterEntityFactory::createCell($this->sanitize($value));
                     }
                 }
             }
         }
         return $cells;
     }
 
+    protected function sanitize(mixed $string): mixed
+    {
+        if (!is_string($string)) {
+            return $string;
```

Comment:
Verständnisfrage: Wenn kein string daher kommt, gibst du den Parmeter einfach wieder zurück.
Ich würde mir hier ein Error-Handling erwarten. Wird sich darum dann zB in der Funktion `createCell` drum gekümmert? Kann da nicht rein klicken.

Reply:
createCell ist von spout und akzeptiert absichtlich mixed. Das macht dann im Excel eine Zahlenspalte wenn es eine Zahl kriegt, und eine Text-Spalte wenn es string kriegt, etc.